packages/main/test/specs/DateTimePicker.spec.js

```diff
@@ -270,12 +270,13 @@ describe("DateTimePicker general interaction", () => {
 		// act
 		await openPickerById("secondaryCalendar");
 		await browser.keys("ArrowUp");
+		await browser.keys("ArrowDown");
 		await browser.keys("Enter");
 		const submitBtn = await getSubmitButton("secondaryCalendar");
 		await submitBtn.click();
 
 		// assert
-		assert.strictEqual(await picker.shadow$("ui5-input").getValue(), "Sha. 17, 1443 AH, 10:27:26 AM", "Value change is applied.");
+		assert.strictEqual(await picker.shadow$("ui5-input").getValue(), "Sha. 24, 1443 AH, 10:27:26 AM", "Value change is applied.");
```

Comment:
This is not testing anything

The value initially is Sha. 24, 1443 AH, 10:27:26 AM

then arrowUp and arrowDown, does not make any new selection.